kratos/tests/test_python_registry.py

```diff
@@ -171,31 +171,31 @@ def testKeys(self):
-            self.assertTrue(item in root_items_keys)
-            if not KratosMultiphysics.Registry.HasValue(item):
-                for subitem in KratosMultiphysics.Registry[item]:
-                    self.assertTrue(subitem in sub_items_keys)
-                    if not KratosMultiphysics.Registry.HasValue(f"{item}.{subitem}"):
-                        for subsubitem in KratosMultiphysics.Registry[f"{item}.{subitem}"]:
-                            self.assertTrue(subsubitem in sub_sub_items_keys)
-                        if not KratosMultiphysics.Registry.HasValue(f"{item}.{subitem}.{subsubitem}"):
-                            for subsubsubitem in KratosMultiphysics.Registry[f"{item}.{subitem}.{subsubitem}"]:
-                                self.assertTrue(subsubsubitem in sub_sub_sub_items_keys)
-
-        # Remove the auxiliary testing entries from the Python registry
-        KratosMultiphysics.Registry.RemoveItem("Processes")
-        KratosMultiphysics.Registry.RemoveItem("PythonRootItem")
+    # #FIXME: This way of checking the iteration will most probably crash once we add more stuff to the registry
+    # #FIXME: Most probably we should check that we're iterating more than one item or something of this sort
+    # def testIteration(self):
+    #     KratosMultiphysics.Registry.AddItem("Processes.KratosMultiphysics.KratosApplication.PythonProcess", KratosMultiphysics.Process())
+    #     KratosMultiphysics.Registry.AddItem("PythonRootItem.PythonSubItem.PythonSubSubItem", object())
+
+    #     root_items_keys = ["Operations","Processes","PythonRootItem"]
+    #     sub_items_keys = ["All","KratosMultiphysics","PythonSubItem"]
+    #     sub_sub_items_keys = ["PythonSubSubItem","KratosApplication","PythonProcess","Operation","Process"]
+    #     sub_sub_sub_items_keys = ["PythonProcess","ModulePath","Prototype","CreateFunction"]
+    #     for item in KratosMultiphysics.Registry:
+    #         self.assertTrue(item in root_items_keys)
+    #         if not KratosMultiphysics.Registry.HasValue(item):
+    #             for subitem in KratosMultiphysics.Registry[item]:
+    #                 self.assertTrue(subitem in sub_items_keys)
+    #                 if not KratosMultiphysics.Registry.HasValue(f"{item}.{subitem}"):
+    #                     for subsubitem in KratosMultiphysics.Registry[f"{item}.{subitem}"]:
+    #                         self.assertTrue(subsubitem in sub_sub_items_keys)
+    #                     if not KratosMultiphysics.Registry.HasValue(f"{item}.{subitem}.{subsubitem}"):
+    #                         for subsubsubitem in KratosMultiphysics.Registry[f"{item}.{subitem}.{subsubitem}"]:
+    #                             self.assertTrue(subsubsubitem in sub_sub_sub_items_keys)
+
+    #     # Remove the auxiliary testing entries from the Python registry
+    #     KratosMultiphysics.Registry.RemoveItem("Processes")
+    #     KratosMultiphysics.Registry.RemoveItem("PythonRootItem")
```

Comment:
what's up with the python tests?

Reply:
This was a shitty test I did to test the iteration of the registry (see my code comment, I was expecting this to fail as soon as we add more stuff to the registry). I think that we should modify this to only iterate, without checking the values, otherwise it would soon become a complete mess to maintain.